app/dashboard/helpers.py

```diff
@@ -51,6 +53,14 @@
 
 logger = logging.getLogger(__name__)
 
+def load_files_in_directory(dir_name):
```

Comment:
I'm not sure this will work in production, since we use S3 buckets to host user generated media

Reply:
This is not a user generated media. Its part of the application. The `assets/wallpapers` directory is where all the banners resides. Its same as loading styles and scripts in the application. I am sure it will work in production. Where we would have had issues is if the users are allowed to upload files, that will be need to handled by S3. But in this case, a PR can be used to add new banners to the application. That's my opinion though.

Reply:
Sorry about that. Let me update the description